src/views/home/topCategoriesByAds/Card/Card.tsx

```diff
@@ -0,0 +1,44 @@
+import ListElement from '../ListElement/ListElement';
+
+type Props = {
+  category: string;
+  amount: number;
+  link: string;
+  products: {
+    product: string;
+    amount: number;
+    link: string;
+  }[];
+};
+
+const Card: React.FC<Props> = ({ category, amount, products, link }) => {
+  return (
+    <div className='w-11/12 sm:w-64 bg-gray-chalk hover:bg-white hover:shadow-gray-400 hover:shadow-xl ease-in duration-300 rounded-lg overflow-hidden'>
+      <a href={link}>
+        <div className='relative'>
+          <img
+            className='h-32 w-full object-cover'
+            src={require(`../../../../assets/images/category/${category}.jpg`)}
```

Comment:
use also relative path, you can just write assets/images/....

Reply:
> You can import image instead of using require.

I can't because the links are dynamic.

> use also relative path, you can just write assets/images/....

It doesn't work because the images are in src folder.
https://stackoverflow.com/questions/37644265/correct-path-for-img-on-react-js